docs/advanced/exporters.md

```diff
@@ -0,0 +1,61 @@
+# About the exporters
+
+The mbed exporters are used to export your code to various third party tools and IDEs. Each exporter implements a `generate` function that produces an IDE specific project file. Exporters benefit from [mbed build tools](https://github.com/ARMmbed/mbed-os/blob/master/docs/BUILDING.md#build-mbed-sdk-library-from-sources). However, instead of using your source and [config data](https://github.com/ARMmbed/mbed-os/blob/master/docs/config_system.md) to create an executable, we use that information to populate an IDE project file that will be configured to build, flash and debug your code. You can find exporter implementations [here](https://github.com/ARMmbed/mbed-os/tree/master/tools/export).
+
+## mbed-cli command
+
+`mbed export -m [target] -i [IDE]`
+
+# Adding export support for a target
+
+If you have added new target to the mbed SDK, exporting will allow users to transition from mbed source code to the offline development environment of their choice. This functionality activates the use of your device for a larger number of users.
```

Comment:
* mbed SDK? Do we still use that term in mbed OS 5? 
* Missing "a" (if you have added a new target)
* "activates the use of your device" is very clunky